src/client/client.c

```diff
@@ -168,7 +168,9 @@ int client_call_manager(Client *client) {
                 return r;
         }
 
-        if (streq(client->op, "list-units")) {
+        if (streq(client->op, "help")) {
```

Comment:
I think it already prints `usage` (and by that `print_client_usage`) for `-h` and `help` here:
https://github.com/containers/hirte/blob/main/src/client/main.c#L27-L31

Reply:
yes, this adds support for `hirtectl help` in addition to `-h` and `--help`